src/main/java/seedu/address/model/Model.java

```diff
@@ -35,14 +35,14 @@ public interface Model {
     void setGuiSettings(GuiSettings guiSettings);
 
     /**
-     * Returns the user prefs' address book file path.
+     * Returns the user prefs' application book file path.
      */
-    Path getAddressBookFilePath();
+    Path getApplicationBookFilePath();
 
     /**
-     * Sets the user prefs' address book file path.
+     * Sets the user prefs' application book file path.
      */
-    void setAddressBookFilePath(Path addressBookFilePath);
+    void setApplicationBookFilePath(Path applicationBookFilePath);
```

Comment:
What is the purpose of changing only these two methods in the original `Model`and `ModelManager`? Why not just use the methods newly created in `ApplicationModel` and `ApplicationModelManager`?

Reply:
These two methods are based off those found in `UserPrefs`. Since the change is pretty simple I just decided to change the names everywhere instead of making a temporary `ApplicationUserPrefs` and `ApplicationReadOnlyUserPrefs`. The remaining methods defined in `Model` will take more effort to change since their signatures changed, so I've left them out for now. Changing those will be the next step.